certificate-service/pdfbox-generator/src/main/java/se/inera/intyg/certificateservice/pdfboxgenerator/pdf/fk7443/FK7443PdfFillService.java

```diff
@@ -23,6 +23,8 @@ public class FK7443PdfFillService implements CertificateTypePdfFillService {
   public static final String PATIENT_ID_FIELD_ID = "form1[0].#subform[0].flt_txtPersonNrBarn[0]";
   public static final String PERIOD_FIELD_ID_PREFIX = "form1[0].#subform[0]";
 
+  private int mcid = 100;
```

Comment:
The service has to be stateless. This introduces state and this integer will continue to grow for each time getAvailableMcid is called. We have to find a solution which doesn't introduce state in the service.

Reply:
Added a fix with AtomicInteger and removed state in the services.